core/modules/widgets/select.js

```diff
@@ -51,7 +51,12 @@ SelectWidget.prototype.render = function(parent,nextSibling) {
 Handle a change event
 */
 SelectWidget.prototype.handleChangeEvent = function(event) {
-	var value = this.getSelectDomNode().value;
+	if(this.selectMultiple == false) {
+		var value = this.getSelectDomNode().value;
+	} else {
+		var value = this.getSelectValues()
```

Comment:
hmmm, ... I know, we have a very relaxed variable declaration handling in TW, but imo this one hurts. see line 55 and 57. @Jermolene ??